testpackage/small_test_carrington_github_ci.sh

```diff
@@ -175,9 +186,9 @@ for run in ${run_tests[*]}; do
    speedup=$( echo $refPerf $newPerf |gawk '{if($2 == $2 + 0 && $1 == $1 + 0 ) print $1/$2; else print "NA"}')
    { {
    echo  "$refPerf        $newPerf         $speedup"
-   echo "------------------------------------------------------------"
-   echo "  variable     |     absolute diff     |     relative diff | "
-   echo "------------------------------------------------------------"
+   echo "-------------------------------------------"
+   echo " variable | absolute diff | relative diff |"
```

Comment:
Oh, that tabulation fix is absolutely great! 
However, I think we also want it on this line so that the absolute and relative diff column headers "align".

Leaving it up to you to decide on how many dashes to include - one option would be to just cover "variable" in the top line, and all the way on the bottom one?